bedrock/firefox/tests/test_base.py

```diff
@@ -84,7 +94,9 @@ def test_one_button_when_channel_specified(self):
         self.client.get(self.url, {
             'channel': 'beta',
         })
-        self.button_mock.assert_called_once_with('beta', force_direct=True,
+        self.button_mock.assert_called_once_with('beta',
+                                                 alt_copy=Markup('Download Now'), button_color='mzp-t-small',
```

Comment:
I like the way this test checks that the button is primary only if there's a single one on the page 👍